staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -772,18 +783,20 @@ func (s *Controller) nodeSyncService(svc *v1.Service) bool {
 // updateLoadBalancerHosts updates all existing load balancers so that
 // they will match the latest list of nodes with input number of workers.
 // Returns the list of services that couldn't be updated.
-func (s *Controller) updateLoadBalancerHosts(ctx context.Context, services []*v1.Service, workers int) (servicesToRetry []*v1.Service) {
+func (s *Controller) updateLoadBalancerHosts(ctx context.Context, services []*v1.Service, workers int) (servicesToRetry map[string]struct{}) {
```

Comment:
I think using sets.String might be better than map[string]struct{}. It would also simplify the tests since you can call [Equal()](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/util/sets/string.go#L156-L161) to compare the list.

Reply:
done.